src/Fixer/Phpdoc/PhpdocTagNoNamedArgumentsFixer.php

```diff
@@ -0,0 +1,195 @@
+    }
+
+    protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
+    {
+        for ($index = $tokens->count() - 1; $index > 0; --$index) {
+            if (!$tokens[$index]->isClassy()) {
+                continue;
+            }
+
+            $prevIndex = $tokens->getPrevMeaningfulToken($index);
+            \assert(\is_int($prevIndex));
+
+            if ($tokens[$prevIndex]->isGivenKind(\T_NEW)) {
+                continue;
+            }
+
+            if (!$this->configuration['fix_attribute_classes'] && self::isAttributeClass($tokens, $prevIndex)) {
+                continue;
+            }
+
+            $this->ensureIsDocBlockWithNoNameArgumentsTag($tokens, $index);
+
+            $docBlockIndex = $tokens->getPrevTokenOfKind($index + 2, [[\T_DOC_COMMENT]]);
+            \assert(\is_int($docBlockIndex));
+
+            $content = $tokens[$docBlockIndex]->getContent();
+
+            $newContent = Preg::replace('/@no-named-arguments.*\R/', rtrim('@no-named-arguments '.$this->configuration['description']).$this->whitespacesConfig->getLineEnding(), $content);
+
+            if ($newContent !== $content) {
+                $tokens[$docBlockIndex] = new Token([\T_DOC_COMMENT, $newContent]);
+            }
+        }
+    }
+
+    private static function isAttributeClass(Tokens $tokens, int $index): bool
+    {
+        while ($tokens[$index]->isGivenKind([\T_FINAL, FCT::T_READONLY])) {
+            $index = $tokens->getPrevMeaningfulToken($index);
+            \assert(\is_int($index));
+        }
+
+        if (!$tokens[$index]->isGivenKind(CT::T_ATTRIBUTE_CLOSE)) {
+            return false;
+        }
+
+        $fullyQualifiedNameAnalyzer = new FullyQualifiedNameAnalyzer($tokens);
+
+        foreach (AttributeAnalyzer::collect($tokens, $tokens->findBlockStart(Tokens::BLOCK_TYPE_ATTRIBUTE, $index)) as $attributeAnalysis) {
+            foreach ($attributeAnalysis->getAttributes() as $attribute) {
+                $attributeName = strtolower($fullyQualifiedNameAnalyzer->getFullyQualifiedName($attribute['name'], $attribute['start'], NamespaceUseAnalysis::TYPE_CLASS));
+                if ('attribute' === $attributeName) {
+                    return true;
+                }
+            }
+        }
+
+        return false;
+    }
+
+    private function ensureIsDocBlockWithNoNameArgumentsTag(Tokens $tokens, int $index): void
+    {
+        /** @var null|callable(WhitespacesFixerConfig, Tokens, int): void $ensureIsDocBlockWithTagNoNameArguments */
+        static $ensureIsDocBlockWithTagNoNameArguments;
+
+        if (null === $ensureIsDocBlockWithTagNoNameArguments) {
+            $ensureIsDocBlockWithTagNoNameArguments = \Closure::bind(
+                static function (WhitespacesFixerConfig $whitespacesConfig, Tokens $tokens, int $index, bool $fixInternal): void {
+                    $phpUnitInternalClassFixer = new PhpUnitInternalClassFixer();
```

Comment:
just to not overlooked as that comment is marked as "outdated"

https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8906#discussion_r2280792403

Reply:
Extracted to [separate PR](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8953) as it changes a couple of files.